src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -313,6 +319,7 @@ public String toString()
                           .add(Option.FAST_PATH.toString(), fastPath)
                           .add(Option.TRANSACTIONAL_MODE.toString(), transactionalMode)
                           .add(Option.TRANSACTIONAL_MIGRATION_FROM.toString(), transactionalMigrationFrom)
+                          .add(PENDING_DROP.name(), pendingDrop)
```

Comment:
other elements use `toString` which lower cases. Also, why are the new accord elements prefixed (except this one)?